lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -78,6 +78,46 @@
                 }
 
             }),
+            TimeZoneFieldView: FieldViews.DropdownFieldView.extend({
+                fieldTemplate: field_dropdown_account_template,
+
+                initialize: function (options) {
+                    this.options = _.extend({}, options);
+                    _.bindAll(this, 'updateList', 'listenToCountryView');
+                    this._super(options);
+                },
+
+                listenToCountryView: function (view) {
+                    this.listenTo(view.model, 'change:country', this.updateList);
+                },
+
+                updateList: function (user) {
+                    var view = this;
+                    $.ajax({
+                        type: 'GET',
+                        url: '/user_api/v1/preferences/time_zones/',
+                        data: {'country_code': user.attributes.country},
+                        success: function (data) {
+                            var countryTimeZones = $.map(data, function (timeZoneInfo) {
+                                return [[timeZoneInfo.time_zone, timeZoneInfo.description]];
```

Comment:
I couldn't find a way to return a list of tuples. http://stackoverflow.com/questions/2917175/return-multiple-values-in-javascript
Any ideas?